core/token.go

```diff
@@ -162,20 +162,19 @@ func (ts TokenService) GetPublicKeyPEM(ctx *HttpContext) (pemPublicKey *rsa.Publ
 	if err := ctx.Request("GET", "/SAAS/API/1.0/REST/auth/token?attribute=publicKey&format=pem", nil, &outp); err != nil {
 		return nil, err
 	}
-	pubKey, err := jwt.ParseRSAPublicKeyFromPEM([]byte(outp))
-	return pubKey, err
+	return jwt.ParseRSAPublicKeyFromPEM([]byte(outp))
 }
 
 /* Validate the ID token (locally). Return the list of claims. */
-func (ts TokenService) ValidateIDToken(ctx *HttpContext, idToken string) (map[string]interface{}, error) {
+func (ts TokenService) ValidateIDToken(ctx *HttpContext, idToken string) error {
```

Comment:
This still seems inconsistent to me. Some errors are handled by printing to ctx.Log.Err and some are encapsulated in a new error object so that the cli code can print it to ctx.Log.Err. All of the other interfaces for cli commands (directory service, apps, oauthresources) log all Info and Err output in the command and return nothing. I'd prefer we go one way (core APIs as just APIs and produce no user output) or the other (core APIs are CLI commands and produce all user output -- like they currently do).